### DISCUSSION TITLE
Favorite Folder & Exports

### DISCUSSION THREAD

**Discussion by clarknova99** [blakeblackshear/frigate#9378] [Ideas]
I would like to be able to watch recordings I flag as favorite in another app like Plex. Currently, I believe I need to individually lookup each favorite via the datetime range on the Exports page. 

It would be a huge time saver if clicking on the favorite star would trigger an export to favorites folder. 
It would then allow me to create a folder mount to favorites that I can also use for streaming via Plex.

**Comment by NickM-27** [comment]
see https://github.com/blakeblackshear/frigate/issues/7710 and leave a 👍

**Reply by clarknova99** [reply]
Thanks for the quick response @NickM-27 ! 

I did take a look at #7710 but it wasn't clear if the favorites were going to be added to the feature export, meaning I would still manually need to go in to export each favorite. 

The intention with this request is to have the export trigger when favoriting an event (clicking on the star).  If that is the intent I'll go in and give it 👍

**Reply by NickM-27** [reply]
Basically the idea is that you would be given an option when favoriting an event to have it be exported
